DuggaSys/duggaed.js

```diff
@@ -649,7 +649,7 @@ function showVariantEditor() {
 function returnedFile(data){
 	retdata = data;
 	filearray = [];
-
+	
```

Comment:
Changes by mistake?

Reply:
yes, i was working on another issue and forgot to change brash ;)